pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.54</version>
+        <version>4.2</version>
```

Comment:
4 uses kotlin runtime, so I would rather stick with 3 to reduce this dependency which could be dangerous in a jenkins world

Reply:
@lanwen 
This is the plugin pom version number not the okhttp3 version number.

Reply:
lol, sorry, wrong look :)